terraform/groups/ecs-service/variables.tf

```diff
@@ -57,107 +35,91 @@ variable "desired_task_count" {
   type        = string
-  default     = "0"
+  description = "The schedule to use when scaling down the number of tasks to zero."
+  # Typically used to stop all tasks in a service to save resource costs overnight.
+  # E.g. a value of '55 19 * * ? *' would be Mon-Sun 7:55pm.  An empty string indicates that no schedule should be created.
+
+  default     = ""
 }
-variable "default_session_expiration" {
+variable "service_scaleup_schedule" {
   type        = string
-  default     = "3600"
+  description = "The schedule to use when scaling up the number of tasks to their normal desired level."
+  # Typically used to start all tasks in a service after it has been shutdown overnight.
+  # E.g. a value of '5 6 * * ? *' would be Mon-Sun 6:05am.  An empty string indicates that no schedule should be created.
+
+  default     = ""
 }
-variable "extensions_web_version" {
-  type        = string
+
+# ----------------------------------------------------------------------
+# Cloudwatch alerts
+# ----------------------------------------------------------------------
+variable "cloudwatch_alarms_enabled" {
+  description = "Whether to create a standard set of cloudwatch alarms for the service.  Requires an SNS topic to have already been created for the stack."
+  type        = bool
+  default     = true
 }
-variable "feature_request_count" {
+
+# ------------------------------------------------------------------------------
+# Service environment variable configs
+# ------------------------------------------------------------------------------
+variable "ssm_version_prefix" {
   type        = string
+  description = "String to use as a prefix to the names of the variables containing variables and secrets version."
+  default     = "SSM_VERSION_"
 }
-variable "human_log" {
-  type        = string
-  default     = "1"
+
+variable "use_set_environment_files" {
+  type        = bool
+  default     = false
+  description = "Toggle default global and shared  environment files"
 }
-variable "log_level" { 
-  type        = string
-  description = "The log level for services to use: trace, debug, info or error"
+variable "log_level" {
   default     = "info"
-}
-variable "maximum_extension_requests_per_day" {
-  type        = string
-  default     = "50"
-}
-variable "max_extension_period_in_months" {
   type        = string
-  default     = "12"
-}
-variable "max_file_size_bytes" {
-  type        = string
-  default     = "4194304"
-}
-variable "permission_name_download" {
-  type        = string
-}
-variable "permission_name_view" {
-  type        = string
-}
-variable "piwik_site_id" {
-  type        = string
-  default     = " 7"
-}
-variable "piwik_url" {
-  type        = string
-}
-variable "session_create" {
-  type        = string
-  default     = "off"
+  description = "The log level for services to use: trace, debug, info or error"
 }
-variable "show_service_unavailable_page" {
-  type        = string
-  default     = "off"
+
+variable "cookie_name" {
```

Comment:
Do we need this terraform variable? This seems not used anywhere. Isn't it?

Reply:
removed